WooCommerce/WooCommerceTests/ViewRelated/Orders/OrderListCellViewModelTests.swift

```diff
@@ -61,4 +61,37 @@ final class OrderListCellViewModelTests: XCTestCase {
         XCTAssertEqual(accessoryView.image, expectedImage)
         XCTAssertEqual(accessoryView.tintColor, .tertiaryLabel)
     }
+
+    func test_salesChannel_when_createdVia_is_pos_rest_api_then_returns_POS() {
+        // Given
+        let order = MockOrders().sampleOrder().copy(createdVia: "pos-rest-api")
+
+        // When
+        let viewModel = OrderListCellViewModel(order: order, currencySettings: ServiceLocator.currencySettings)
+
+        // Then
+        XCTAssertEqual(viewModel.salesChannel, "POS")
```

Comment:
nit: is "POS" directly shown in the UI? If so, might we consider making this string localizable?

Reply:
Good point, it does show in the UI but I didn't consider it's a string we'd make localizable, ie: In Spanish the acronym is still `POS` despite the translation being _"punto de venta"_ 🤔 . I see this is set as localizable string in `POSTabViewController` thought and definitely makes sense for it to be localized. Updated on https://github.com/woocommerce/woocommerce-ios/pull/15848/commits/1f93588f14cf61e2118ad4f4e0c87ae393dcf551